common/core/web/keyboard-processor/package.json

```diff
@@ -19,10 +19,11 @@
   "homepage": "https://github.com/keymanapp/keyman#readme",
   "devDependencies": {
     "@keymanapp/resources-gosh": "^15.0.48",
-    "chai": "^4.2.0",
-    "mocha": "^5.2.0",
-    "mocha-teamcity-reporter": "^3.0.0",
-    "typescript": "^3.8.3"
+    "chai": "^4.3.4",
+    "mocha": "^8.4.0",
+    "mocha-teamcity-reporter": "^4.0.0",
+    "typescript": "^3.8.3",
+    "ts-node": "^9.1.1"
```

Comment:
Good to see `ts-node` move to a dev dependency

Reply:
Yep.  Wasn't as sure what its role was when I first made the package, so it was a pretty simple mistake to correct.